tests/x509/test_ocsp.py

```diff
@@ -113,3 +115,20 @@ def test_invalid_serialize_encoding(self):
             req.public_bytes("invalid")
         with pytest.raises(ValueError):
             req.public_bytes(serialization.Encoding.PEM)
+
+
+def test_create_ocsp_request_from_cert():
+    from cryptography.hazmat.backends.openssl.backend import backend
+    cert = _load_cert(
+        os.path.join("x509", "cryptography.io.pem"),
+        x509.load_pem_x509_certificate,
+        backend
+    )
+    issuer = _load_cert(
+        os.path.join("x509", "rapidssl_sha256_ca_g3.pem"),
+        x509.load_pem_x509_certificate,
+        backend
+    )
+    req = ocsp.create_ocsp_request_from_cert(cert, issuer, hashes.SHA1())
+    with open("/Users/pkehrer/Desktop/out.der", "wb") as f:
```

Comment:
this is a cool unit test

Reply:
Yep :) I have it doing something different locally now but didn't see much point to pushing it when I know we'll be going through other revisions